composer.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "metamodels/boilerplate",
+  "description": "MetaModels boilerplate",
+  "keywords": [
+    "contao",
+    "metamodels",
+    "core",
+    "boilerplate"
+  ],
+  "type": "contao-module",
+  "homepage": "http://now.metamodel.me/",
+  "license": "LGPL-3.0+",
+  "authors": [
+    {
+      "name": "Andreas Isaak",
+      "email": "info@andreas-isaak.de",
+      "role": "Developer"
+    },
+    {
+      "name": "Ingolf Steinhardt",
+      "email": "info@e-spin.de",
+      "role": "Developer"
+    }
+  ],
+  "support": {
+    "email": "mail@metamodel.me",
+    "issues": "https://github.com/MetaModels/boilerplate/issues",
+    "wiki": "http://de.contaowiki.org/MetaModels",
+    "irc": "irc://irc.freenode.org/contao.mm",
+    "source": "https://github.com/MetaModels/boilerplate"
+  },
+  "require": {
+    "php": ">=5.4",
+    "contao/core": "^3.5.5",
+    "contao-community-alliance/composer-plugin": "~2.0",
+    "metamodels/core": "~2.0",
+  },
+  "require-dev": {
+    "cyberspectrum/contao-toolbox": "~0.6",
+    "phpcq/all-tasks": "~1.1",
+    "metamodels/base-unit-tests": "~1.0",
+    "metamodels/phpunit-contao-database": "~1.0"
+  },
+  "conflict": {
+    "contao/core": "3.4.0"
+  },
+  "autoload": {
+    "psr-0": {
+      "MetaModels": "src"
+    }
+  },
+  "autoload-dev": {
+    "psr-0": {
```

Comment:
it´s a copy of https://github.com/MetaModels/core/blob/master/composer.json ... I´m not sure if is necessary...